### PR TITLE
Makes vox footprints look less awful, etc.

### LINKED ISSUE #9431: New vox footprints look bad and also dumb
The claws are too long and spread too wide, and the "heel" should extend back just a bit
Also they should look like human footprints while wearing shoes, though I'm not sure that's possible

### PR DESCRIPTION
**Pull request by Exxion**
First and most obviously, this changes the sprite of vox footprints to something more appropriate. It's not great, but it's better than before.
image
<sub>Fuck, there's a cursor in that image. I'd retake it but I already shut down Dream Daemon, so oh well</sub>

Also, footprints now work a bit differently. If you are wearing shoes, the appearance of your footprint depends on the shoe rather than your foot. (Currently, all shoes except the vox boots use the standard footprint we're all used to.) If you are not wearing shoes, it uses your species' footprint type. (Currently, all species except vox use the standard footprint type.)

It would now be extremely easy to add barefoot footprint types for other races, as well as footprint types for different shoes (clown shoes come to mind), but this PR doesn't do it. Also, it would now be easier to change footprints for people without legs or with prosthetic legs, though not nearly as easy as the previous thing.

Also, this fixes an old and (as far as I can tell) unreported bug that caused people to not track blood if they were not wearing shoes.

Fixes #9431

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
Looks nicer

**Comment by Intigracy** [comment]
fyi these footprints have always worked for traders iirc, I just fixed the check so it worked for vox who weren't traders.

**Comment by Exxion** [comment]
I'd never seen them before

**Comment by SangriaMonkey** [comment]
when

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1719,3 +1719,7 @@
 		return 1
 
 	return 0
+
+/mob/living/carbon/human/proc/get_footprint_type()
+	var/obj/item/clothing/shoes/S = shoes //Why isn't shoes just typecast in the first place?
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/human/human.dm]: pretty sure there's a wear_shoes or something or other

**Comment by clusterfack** [comment]
Not only that you've mastered OOP better than the majority of the PRs I've seen today with just a few lines of code

**Pull request merged**